model/org.eclipse.birt.report.model/src/org/eclipse/birt/report/model/api/DesignElementHandle.java

```diff
@@ -571,7 +569,7 @@ public void clearProperty(String propName) throws SemanticException {
 	 */
 
 	public void clearAllProperties() throws SemanticException {
-		List props = getDefn().getProperties();
+		List<?> props = getDefn().getProperties();
```

Comment:
Here too, type info is being lost.  

Something like this would seem better:
```
	public void clearAllProperties() throws SemanticException {
		List<IElementPropertyDefn> props = getDefn().getProperties();

		ActivityStack stack = module.getActivityStack();
		stack.startTrans(CommandLabelFactory.getCommandLabel(MessageConstants.CLEAR_PROPERTIES_MESSAGE));

		try {
			for (IElementPropertyDefn propDefn : props) {
				String propName = propDefn.getName();
```

Reply:
Typ info is lost: before it was "List", now it is "List<?>" where is the type lost?

I will try your suggestion.

Reply:
Change is done.